src/fis/core/ingest.py

```diff
@@ -174,9 +174,13 @@ async def populate_by_event(
 
     async def store_secret(self, *, file_secret: str) -> str:
         """Communicate with HashiCorp Vault to store file secret and get secret ID"""
+        log.debug("Storing secret...")
         try:
             return self._vault_adapter.store_secret(secret=file_secret)
         except self._vault_adapter.SecretInsertionError as error:
             comms_error = VaultCommunicationError(message=str(error))
             log.error(comms_error)
             raise comms_error from error
+        except Exception as general_error:
+            log.error("Error in the secret insertion %s", general_error)
+            raise general_error
```

Comment:
Shouldn't these uncaught exceptions show up on the console anyhow?

Reply:
Should be, but I put them just to be sure. This is a temporary branch for debugging, will be deleted without merging anyway.